auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -140,7 +140,8 @@ public void onClick(View view) {
                     }
                 });
                 provider.setAuthenticationCallback(this);
-                btnHolder.addView(loginButton, 0);
+                // Account for the email provider which should always be at the bottom
+                btnHolder.addView(loginButton, btnHolder.getChildCount() - 1);
```

Comment:
Now that I think about this, do we really want the email provider to always be at the bottom? Why aren't we using the supplied order from the dev?